e2e/support/helpers/e2e-custom-column-helpers.js

```diff
@@ -1,3 +1,7 @@
+export function expressionEditorWidget() {
+  return cy.findByTestId("expression-editor").should("exist");
```

Comment:
it will fail if it doesn't exist. also you're using this function as a selector, no need to put additional assert inside in this case

```suggestion
  return cy.findByTestId("expression-editor");
```